server/controllers/routes/admin/cohort/editCohort.js

```diff
@@ -0,0 +1,26 @@
+const { putSpecificCohort } = require('../../../../database/queries');
+const { editCohortSchema } = require('../../../../validation/index');
+
+const editCohort = async (req, res, next) => {
+  try {
+    req.body.cohortId = req.params.cohortId;
+    await editCohortSchema.validate(req.body, { abortEarly: false });
+    const result = await putSpecificCohort(req.body);
+    if (result.rowCount === 1) {
+      res.json({ statusCode: 200, message: 'Changed Succefully' });
+    } else {
+      res.status(404).json({
+        statusCode: 404,
```

Comment:
@Mu7ammadAbed You could remove statusCode from the json body because you already called `status(404)` on the response.